src/openvic-simulation/types/Sliders.hpp

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "openvic-simulation/types/fixed_point/FixedPoint.hpp"
+#include "openvic-simulation/utility/Getters.hpp"
+
+namespace OpenVic {
+  // TODO: make struct for tariff slider, unlike other sliders, you can have negative values
+  struct slider_value_t {
+  friend struct  SliderManager;
+  private:
+      int PROPERTY(min);
```

Comment:
Best to use `int32_t` rather than `int`, just to be certain about the data type. You could even define a type alias `using int_type = int32_t` and then just use `int_type` throughout the struct which could be useful if we ever need to change the type for any reason